src/ResourceManagement/Network/Microsoft.Azure.Management.Network/Generated/ApplicationGatewaysOperations.cs

```diff
@@ -1346,6 +1346,224 @@ internal ApplicationGatewaysOperations(NetworkManagementClient client)
         }
 
         /// <summary>
+        /// The BackendHealth operation gets the backend health of application gateway
+        /// in the specified resource group through Network resource provider.
+        /// </summary>
+        /// <param name='resourceGroupName'>
+        /// The name of the resource group.
+        /// </param>
+        /// <param name='applicationGatewayName'>
+        /// The name of the application gateway.
+        /// </param>
+        /// <param name='expand'>
+        /// expand references resources.
+        /// </param>
+        /// <param name='customHeaders'>
+        /// The headers that will be added to request.
+        /// </param>
+        /// <param name='cancellationToken'>
+        /// The cancellation token.
+        /// </param>
+        public async Task<AzureOperationResponse<ApplicationGatewayBackendHealth>> BackendHealthWithHttpMessagesAsync(string resourceGroupName, string applicationGatewayName, string expand = default(string), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
I am not able to see test coverage for the newly added API's. Please add test coverage.